hosts/darwin02/builder.nix

```diff
@@ -0,0 +1,25 @@
+{
+  users.knownGroups = [ "nix" ];
+  users.knownUsers = [ "nix" ];
+
+  users.groups.nix = {
+    name = "nix";
+    gid = 8765;
+    description = "Group for remote build clients";
+  };
+
+  users.users.nix = {
```

Comment:
Now that it's deployed I don't know if it's worth changing it, but in srvos we named that user "nix-remote-builder" to clarify the role of that user. `nix` might be mistaken as the user for the nix-daemon.

Reply:
I used `nix` here for consistency with our existing builder config but this would also be addressed by using the srvos module.